ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/event-handlers.yml

```diff
@@ -45,6 +45,12 @@ event_handlers:
       regex: "ab*"
     actions:
       increment_device_health_metric: Metric2
+  - name: trigger-on-logging2
```

Comment:
Will you please take on look for trigger on-maintenance 
Generating invalid config for below input
```
- name: mani
    trigger: on-maintenance
    trigger_on_maintenance:
      operation: enter
      action: all
```
trigger on-maintenance enter all

Reply:
Fixed